Documentation/xcassets/swift2.md

```diff
@@ -21,6 +21,8 @@ You can customize some elements of this template by overriding the following par
 | Parameter Name | Default Value | Description |
 | -------------- | ------------- | ----------- |
 | `enumName` | `Asset` | Allows you to change the name of the generated `enum` containing all assets. |
+| `colorTypeName` | `ColorAsset` | Allows you to change the name of the struct type representing a color. |
+| `imageTypeName` | `ImageAsset` | Allows you to change the name of the struct type representing an image. |
 | `noAllValues` | N/A | Setting this parameter will disable generation of the `allValues` constant. |
```

Comment:
I don't know, it's definitely handy when you need it, and it shouldn't matter how big the generated file is. Constants as these are only allocated when accessed, so that shouldn't be an issue either.